x-pack/plugins/security/public/management/roles/edit_role/privileges/kibana/feature_table/sub_feature_form.tsx

```diff
@@ -33,14 +40,37 @@ export const SubFeatureForm = (props: Props) => {
     .getPrivilegeGroups()
     .filter((group) => group.privileges.length > 0);
 
+  const tooltip = useMemo(() => {
```

Comment:
Just a performance optimisation to avoid unnecessary renders.

Reply:
I would try and avoid blind performance optimisations since it's not guaranteed to be faster and could lead to unexpected behaviours / difficult to debug race conditions. Memoization trades CPU time with memory allocation. I'm currently not sure that tradeoff is worth it in this case unless we've measured the impact.

Reply:
Got it, I'm ok removing `useMemo` then. Thanks for the feedback!

Reply:
@thomheymann https://github.com/elastic/kibana/pull/143733/commits/49d634cef984419c66714729a63b0af98faec233